setup.py

```diff
@@ -1,9 +1,12 @@
-import sys
+#!/usr/bin/env python
```

Comment:
I prefer no shebang here, too. It doesn't help anything anyway if you haven't made the file executable, and by convention I've only ever seen setup.py files invoked as `python setup.py`.